support-frontend/assets/pages/[countryGroupId]/router.tsx

```diff
@@ -27,7 +26,7 @@ const router = createBrowserRouter(
 		},
 		{
 			path: `/${geoId}/one-time-thank-you`,
-			element: <OneTimeThankYou geoId={geoId} appConfig={appConfig} />,
+			element: <ThankYou geoId={geoId} appConfig={appConfig} />,
```

Comment:
Why do we have two routes if they go to the same component now?

Reply:
Good point, have simplified/removed `one-time-thankyou` and reused the `thank-you`